axelor-business-project/src/main/resources/views/ProjectTask.xml

```diff
@@ -118,7 +118,9 @@
   <form name="business-project-task-form" title="Task"
     model="com.axelor.apps.project.db.ProjectTask"
     onNew="action-business-project-project-task-group-on-new"
-    onLoad="action-business-project-project-task-group-on-load" width="large">
+    onLoad="action-business-project-project-task-group-on-load"
+    onSave="action-sprint-validate-upadte-or-create-project-plannings,action-project-task-method-update-project-plannings"
```

Comment:
Don't need to generate or update planification at save, just need it at sprint onchange in sprint-project-task-form